sql/mob_spawn_mods.sql

```diff
@@ -232,6 +232,8 @@ INSERT INTO `mob_spawn_mods` VALUES (16916817,49,1,1);
 
 INSERT INTO `mob_spawn_mods` VALUES (17326081,50,25,1); -- Angra_Mainyu chainspell at 25%
 
+INSERT INTO `mob_spawn_mods` VALUES (17428812,17,1,1); -- Habetrot's Rumble Crawler

```

Comment:
The no_despawn is for the Rumble Crawler that spawns in place of Habetrot, not Habetrot himself.

I'll add a gil mod to Habetrot.